website/docs/language/checks/index.mdx

```diff
@@ -43,7 +43,9 @@ You can use a scoped data source to validate the status of a piece of infrastruc
 
 #### Meta-Arguments
 
-Scoped data sources support the `depends_on` and `provider` [meta-arguments](/terraform/language/resources/syntax#meta-arguments). Scoped data sources do not support the `count` or`for_each` meta-arguments.
+Meta-arguments are arguments built into Terraform that control how Terraform creates and manages resources. [Meta-arguments](/terraform/language/meta-arguments) for additional information.
```

Comment:
Can you take this change out? This is the page I'm replacing and I don't want you to have a merge conflict!